README.md

```diff
@@ -163,15 +173,21 @@ Visit http://localhost:8000 on your browser to access the API documentation.
 
 ## Setup Frontend
 
-- Create a file named **.env** in the frontend root folder (same directory with package.json)
-- Copy the content of **.env.example** and paste into the new .env file.
 - On your terminal/command line, navigate to **./zubhub/zubhub_frontend/zubhub** directory
 
+- Create a file named **.env** in the frontend root folder (same directory with package.json), then copy-past the following lines
```

Comment:
Typo: `copy-paste` 

Maybe we can make this sentence as,
```
Create a file named **.env** in the frontend root folder (same directory with package.json), with the following content
```

Reply:
Done